contracts/vault/packages/VaultPackage.sol

```diff
@@ -70,6 +69,9 @@ contract VaultPackage is VaultStorage, IVault, IVaultInit, IVaultEvents {
     /// @notice Set the new accountant address.
     /// @param newAccountant The new accountant address.
     function setAccountant(address newAccountant) external override onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (newAccountant == address(0)) {
```

Comment:
I don't think that's the case because on initialize the accountant can be zero. So we still need to check this on _assessFees or add a check on initialize function to revert if accountant is zero address.